files-community/Files
# RichCommands: SearchUnindexedItems

**Resolved / Related Issues**
   Related #11481 

**Validation**
How did you test these changes?
   1. Open app
   2. Search something that is not indexed
   3. Click on `Search unindexed items`
